awx/main/access.py

```diff
@@ -411,8 +415,8 @@ def check_org_host_limit(self, data, add_host_name=None):
                 )
             )
 
-        if add_host_name:
-            host_exists = Host.objects.filter(inventory__organization=org.id, name=add_host_name).exists()
+        if kwargs.get('add_host_name', None):
```

Comment:
`.get(...)` automatically falls back to providing `None` when the key is not in the dict.